### PR TITLE
Improve and fix Add Site errors

### PR DESCRIPTION
**Pull request by rmccue**
* Refactors the form handler into something more readable
* Adds validation for domains and paths (reserved paths now blocked, domains validated properly)
* Fixes up some JS problems
* Fixes issue with branding CSS
* Adds placeholders to indicate valid inputs

altis-v3-add-site

Built off #93.

### CODE REVIEW

File: assets/customize-settings.js [hunk 1]
```diff
@@ -1,43 +1,52 @@
...
+( function () {
+	/*
+	* Custom JS to modify the Altis CMS module Add New Site page
+	*/
+	const siteSubdomain = document.getElementById( 'site-subdomain' );
+	const siteSubdirectory = document.getElementById( 'site-subdirectory' );
+	const siteCustomDomain = document.getElementById( 'site-custom-domain' );
+	const siteAddress = document.getElementById( 'site-address' );
+	const siteAddressDesc = document.getElementById( 'site-address-desc' );
+	const networkAddress = window.location.hostname;
```
Comment (roborourke) [inline assets/customize-settings.js]: Can the network admin be accessed via subsite or always on the primary hostname?

File: inc/branding/namespace.php [hunk 2]
```diff
@@ -17,7 +17,7 @@
  */
 function bootstrap() {
 	add_action( 'add_admin_bar_menus', __NAMESPACE__ . '\\remove_wordpress_admin_bar_item' );
-	add_action( 'admin_enqueue_scripts', __NAMESPACE__ . '\\enqueue_admin_scripts' );
+	add_action( 'admin_bar_init', __NAMESPACE__ . '\\enqueue_admin_scripts' );
```
Comment (roborourke) [inline inc/branding/namespace.php]: 😱 picking up my bad habits!

Comment (roborourke) [reply inc/branding/namespace.php]: Also the admin bar can be turned off

File: assets/customize-settings.js (hunk 1 above)
Comment (rmccue) [reply assets/customize-settings.js]: Yes, but I think this matches what WordPress already does? In any case, I didn't change this code, just refactored.

File: inc/branding/namespace.php (hunk 2 above)
Comment (rmccue) [reply inc/branding/namespace.php]: This is actually intentionally like this, as it needs to enqueue on the frontend as well. The change in here broke the Altis logo in the admin bar on the frontend.

**Pull request merged**